source/styleEditor.js

```diff
@@ -1,3 +1,10 @@
 $(document).ready(function(){
-  // your code goes here.
+  $('#style_editor').on('submit', function(){
+    let selector = $("[name=selector]").val();
+    // alert(selector);
+    let property = $("[name=property]").val();
+    let value    = $("[name=value]").val();
+    $(selector).css(property, value);
+  event.preventDefault();
```

Comment:
When using `event.preventDefault()`, make sure to pass `event` as an argument to your anonymous function on line 2.